doc/man3/SSL_CTX_sess_set_get_cb.pod

```diff
@@ -43,9 +43,9 @@ L<SSL_CTX_set_session_cache_mode(3)>).
 (SSL/TLS server only.)
 
 SSL_CTX_sess_get_new_cb(), SSL_CTX_sess_get_remove_cb(), and
-SSL_CTX_sess_get_get_cb() allow to retrieve the function pointers of the
-provided callback functions. If a callback function has not been set,
-the NULL pointer is returned.
+SSL_CTX_sess_get_get_cb() allow retrieval of the function pointers
+provided to the associated set callback functions. If a callback function
```

Comment:
Shouldn't "provided" be removed?  The sentence doesn't make sense to me for the moment.

Reply:
I think it is worse without the provided.  I've reworded differently.